content/specific-skill-success-criteria/for-loops/_index.md

```diff
@@ -96,11 +96,11 @@ for (let n=0; n<3; n++){
 }
 ```
 
-## All learners should be able to iterate over strings and arrays.
+### All learners should be able to iterate over strings and arrays.
 
 Some people think you need to turn strings into arrays before you can start iterating over them. This is not the case!
 
-## All learners should understand the basics of while loops.
+### All learners should understand the basics of while loops.
```

Comment:
In most files, headings dont end with a full stop. Here they do. Can it be made more consistent?